pxr/usd/bin/common.cpp

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright 2022 Apple
+//
+// Licensed under the Apache License, Version 2.0 (the "Apache License")
+// with the following modification; you may not use this file except in
+// compliance with the Apache License and the following modification to it:
+// Section 6. Trademarks. is deleted and replaced with:
+//
+// 6. Trademarks. This License does not grant permission to use the trade
+//    names, trademarks, service marks, or product names of the Licensor
+//    and its affiliates, except as required to comply with Section 4(c) of
+//    the License and to reproduce the content of the NOTICE file.
+//
+// You may obtain a copy of the Apache License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the Apache License with the above modification is
+// distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the Apache License for the specific
+// language governing permissions and limitations under the Apache License.
+//
+
+#include "common.h"
+#include <pxr/pxr.h>
+
+#include <iostream>
+#include <fstream>
+
+
+std::string GetVersionString() {
+    return std::to_string(PXR_MAJOR_VERSION) + "." + std::to_string(PXR_MINOR_VERSION) + "." + std::to_string(PXR_PATCH_VERSION);
+}
+
+int GetFileSize(const std::string &path) {
```

Comment:
Probably `ArchGetFileLength(...)` would work instead of adding a new function?

It uses GetFileSize on Windows, and fstat for linux/mac. I think that's okay here.

Reply:
Ah good call, I was looking for `FileSize` and couldn't find a function.
Swapped it out in the latest commit